syntax = "proto3";
package pb;

option go_package = "github.com/ardanlabs/python-go/grpc/pb";

message Object_Omni{
   uint32 id = 1;
   uint32 x = 2;
   uint32 y = 3;
   uint32 dist = 4;
   uint32 conf = 6;
}

message Request_Omni_Calib {
     bool check = 1;
}

message Response_Omni {
   bytes omni = 1;
   uint32 img_to_send = 2;
   repeated Object_Omni objects = 3; // list of 3 bytes 'classe' 'x' 'y'
   
}

service Yolo_Omni {
   rpc Send_Omni(Request_Omni_Calib) returns (stream Response_Omni) {}
}

